### ISSUE TITLE
[FR] Can I add shortcut key functions?

### ISSUE THREAD

**Issue by h1533** [hiddify/hiddify-next#422]
### Feature description

Please add a one-click to open and close the proxy shortcut.

**Comment by snsnsjsn** [comment]
yes I can do it  in clash verge to open panel, but the clash verge with webview have problem, so I switch to Hiddify ,but I could not found somewhere to add shortcut
